crux-mir-comp/test/symb_eval/comp/munge_struct.rs

```diff
@@ -0,0 +1,20 @@
+extern crate crucible;
+use crucible::*;
+use crucible::cryptol::munge;
+
+#[derive(Clone)]
+struct Point{
+    x: u32,
+    y: u32,
+}
+
+fn reflect(p: Point) -> Point {
+    Point{x: p.y, y: p.x}
+}
+
+#[crux_test]
+fn munge_struct_equiv_test() {
+    let (x, y) = <(u32, u32)>::symbolic("p");
+    let p2 = reflect(munge(Point{x, y}));
```

Comment:
Oops, the munge might be better placed outside of reflect. Either way the test tests that it doesn't crash crux-mir-comp, but this way it's a little weird looking.

Reply:
Should work either way, just as long as the argument is a struct

Reply:
Ok